src/init.luau

```diff
@@ -1,6 +1,6 @@
 --[[
 	A module to work with detecting devices.
-	Programmed by break-core.
+	Programmed by break-core. Contributed by VSCPlays
```

Comment:
I don't think there's a reason to have this here. Your GitHub name will show up on the contributors list once this goes through.

Reply:
I might be a little biased on this though... but it is kind of redundant to have this here.

Reply:
removed it